infrastructure/src/main/java/org/corfudb/infrastructure/BatchProcessor.java

```diff
@@ -135,16 +129,16 @@ private void processor() {
                     currOp.getFutureResult().completeExceptionally(
                             new QuotaExceededException("Quota of "
                                     + streamLog.quotaLimitInBytes() + " bytes"));
-                    log.warn("batchprocessor: quota exceeded, dropping msg {}", currOp.getMsg());
-                } else if (currOp.getType() == Type.SEAL && currOp.getMsg().getEpoch() >= sealEpoch) {
+                    log.warn("batchWriteProcessor: quota exceeded, dropping msg {}", currOp.getMsg());
```

Comment:
It seems like these changes were based on an older master, undo this file's changes?